ROMFS/px4fmu_common/init.d/airframes/4015_holybro_s500

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh
-#
-# @name Holybro S500
```

Comment:
I'm not sure about this one. I mean it's hardly worth more than "generic quad", but if anyone has this airframe selected and it disappears its a terrible user experience (parameters lost, etc).

Reply:
Fair point. I've reverted the removal.

Going forward we really only should allow airframe configs into main that have a huge benefit for the whole community, and will likely be supported for a long time. Otherwise we end up with a big graveyard with airframes that people keep using but are not maintained.
I much prefer to let users select a generic airframe and then do the output mapping themselves, ideally with a section in the user guide about the specif airframe bringup.